tools/setenv.sh

```diff
@@ -62,9 +74,17 @@ zpp_nasm() {
 
 _zpp_build_nasm() {
   local nasm_file=$1
+  local zpp_lib_name_noext=$(basename $ZPP_LIB_NAME .so)
+  local zpp_lib_name_nolibprefix=$(echo $zpp_lib_name_noext | sed 's/lib//g')
   mkdir -p $ZPP_ARTIFACTS_PATH
   zpp_nasm $nasm_file -o $ZPP_ARTIFACTS_PATH/a.o
-  LD_LIBRARY_PATH="" ld $ZPP_ARTIFACTS_PATH/a.o -o $ZPP_ARTIFACTS_PATH/a.out
+  LD_LIBRARY_PATH=" "
+  ld $ZPP_ARTIFACTS_PATH/a.o \
+    -L$ZPP_LIB64_PATH \
+    -l$zpp_lib_name_nolibprefix \
+    -o $ZPP_ARTIFACTS_PATH/a.out \
+    -dynamic-linker /lib64/ld-linux-x86-64.so.2 \
```

Comment:
the default linker was not working for me so I had to specify it, do you have an idea to resolve dynamically the linker?